Gemfile

```diff
@@ -85,6 +85,12 @@ group :development do
   gem 'hint-rubocop_style'
 end
 
+group :test do
+  gem 'capybara', '>= 2.15'
+  gem 'selenium-webdriver'
+  gem 'chromedriver-helper'
```

Comment:
While writing a blog post about this I realized that we don't need the `chromedriver-helper` gem

Reply:
Since we are using the `selenium/standalone-chrome` container.